demos/mqtt/mqtt_demo_lightweight/mqtt_demo_lightweight.c

```diff
@@ -261,6 +261,18 @@ static uint16_t getNextPacketIdentifier( void );
 static uint32_t calculateElapsedTime( uint32_t later,
                                       uint32_t start );
 
+/* / ** */
+/*  * @brief Attempts to resubscribe to topic filter after server refusal */
+/*  * (decoded from SUBACK). */
+/*  * */
+/*  * @param[in] pNetworkContext Pointer to the network context created using Plaintext_Connect. */
+/*  * @param[in] pFixedBuffer Pointer to a structure containing fixed buffer and its length. */
+/*  * The buffer is used for serializing SUBSCRIBE packet, deserializing incoming MQTT packet. */
+/*  * */
+/*  * / */
+/* static void handleResubscribe( NetworkContext_t * pNetworkContext, */
+/*                                        MQTTFixedBuffer_t * pFixedBuffer ); */
+
```

Comment:
Accidental commit?

Reply:
Yes sorry, the most recent commit was incorrect. Updating.